cells/prelude/devshellProfiles.nix

```diff
@@ -0,0 +1,47 @@
+{
+  inputs,
+  cell,
+}: let
+  # explicitly use latest index to avoid warning
+  inherit (cell.library) haskell-nix last-index-state;
+in {
+  mkDev = project: _: let
+    inherit (project.args) compiler-nix-name;
+  in {
+    imports = [
+      (haskell-nix.haskellLib.devshellFor project.shell)
+    ];
+
+    commands = [
+      {
+        package = haskell-nix.cabal-install.${compiler-nix-name};
+        name = "cabal";
+        category = "development";
+      }
+      {
+        package = haskell-nix.tool compiler-nix-name "ghcid" {
+          version = "latest";
+          index-state = last-index-state;
```

Comment:
if you don't pass this argument `haskell.nix` will just use the latest index-state. I think the whole `last-index-state` thing is unnecessary.

Reply:
it is unnecessary except that being explicit about it remove a warning (repeated of each tool).